src/features/courses/course-repository.ts

```diff
@@ -445,6 +446,25 @@ class CourseRepository {
             }
         });
     }
+
+    async updateGrade(options: UpdateGradeOptions): Promise<UpdateResult<StudentGrade>> {
+        try {
+            const updates = await StudentGrade.update(options.updates, {
+                where: {
+                    ...options.where,
+                    active: true
+                },
+                returning: true,
+            });
+            return {
+                updatedCount: updates[0],
```

Comment:
Do we need to document the mapping of these array returns?

Reply:
Not sure what you're asking, but:
1. The response is typed
2. This is not actually an array, it's a tuple